packages/cli/src/sso/saml/routes/saml.controller.ee.ts

```diff
@@ -126,10 +126,11 @@ export class SamlController {
 				}
 			}
 			if (loginResult.authenticatedUser) {
-				void this.internalHooks.onUserLoginSuccess({
+				this.eventRelay.emit('user-logged-in', {
```

Comment:
Just to make sure, these didn't produce telemetry?

Reply:
All good, it [does not](https://github.com/n8n-io/n8n/blob/292bbb3f9f9fe1fbf87ff1ff8531bd08d76cc48e/packages/cli/src/InternalHooks.ts#L851-L864). Refer to the PR description:

> Internal hooks that existed only for the event bus have been deleted, as they are now covered by events: `onNodeBeforeExecute`, `onNodePostExecute`, `onWorkflowBeforeExecute`, `onUserLoginSuccess`, `onUserLoginFailed`